tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -1374,6 +1383,8 @@ IncludeByKey.SSP1_ENABLED {
 				name	"$SSP1_PCM_NAME"
 				id 1
 				direction	"duplex"
+				playback_pause_supported "true"
+				capture_pause_supported "true"
```

Comment:
if we use the hda_force_pause_support kernel parameter we don't need to special-case the nocodec topology.

Best to define the token as false always, and gradually re-add support as 'true' if/when ready on a platform-by-platform basis